src/Nancy.Swagger/Standalone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Nancy.Swagger.Services;
+using Nancy.TinyIoc;
+using Swagger.ObjectModel;
+
+namespace Nancy.Swagger
+{
+    public static class Standalone
+    {
+        public static SwaggerRoot Generate(TinyIoCContainer container, NancyContext context)
```

Comment:
You are correct about the container - I refactored this so that there is only one Resolve() call left here, so this is really unnecessary now.

I really want to keep this functionality out of the demo, so that it is actually usable to clients of the Nancy.Swagger package. Without this class, I wouldn't need to add any code to the library at all, and this could just be a Wiki page.